SPECS/rubygem-rexml/rubygem-rexml.spec

```diff
@@ -30,10 +30,13 @@ gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-
 
 %files
 %defattr(-,root,root,-)
-%doc %{gemdir}/gems/%{gem_name}-%{version}/LICENSE.txt
+%license %{gemdir}/gems/%{gem_name}-%{version}/LICENSE.txt
```

Comment:
you can remove %{gemdir}/gems/%{gem_name}-%{version} similar to what I did for my [PR here](https://github.com/microsoft/azurelinux/pull/8641/files).
If the LICENSE.txt is in the source folder, you can just add that LICENSE file.